plugins/modules/notification/discord.py

```diff
@@ -0,0 +1,168 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright: (c) 2021, Christian Wollinger <cwollinger@web.de>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+DOCUMENTATION = '''
+---
+module: discord
+short_description: Send discord messages
+description:
+  - Send a messages to a discord channel via the discord webhook API.
+author: Christian Wollinger (@cwollinger)
+notes:
+  - "Find the API documentation for Discord API here: U(https://discord.com/developers/docs/resources/webhook#execute-webhook)."
+options:
+  webhook_id:
+    description:
+      - "Format from discord webhook URL: C(/webhooks/{webhook.id}/{webhook.token})."
```

Comment:
```suggestion
      - The webhook ID.
      - "Format from discord webhook URL: C(/webhooks/{webhook.id}/{webhook.token})."
```
(same for `webhook_token`)